AuthenticatorShared/UI/Vault/ItemList/ItemList/ItemListProcessor.swift

```diff
@@ -499,9 +500,13 @@ extension ItemListProcessor: AuthenticatorKeyCaptureDelegate {
                 totpKey: key,
                 username: nil
             )
-            try await services.authenticatorItemRepository.addAuthenticatorItem(newItem)
-            state.toast = Toast(text: Localizations.verificationCodeAdded)
-            await perform(.refresh)
+            if sendToBitwarden {
+                await moveItemToBitwarden(item: newItem)
```

Comment:
We don't show a toast in this case because moveToBitwarden actually launches and foregrounds the PM app. So the toast in the Auth app would't be seen.